pkgs/development/haskell-modules/hackage-packages.nix

```diff
@@ -152448,7 +152448,6 @@ self: {
        description = "hOpenPGP-based command-line tools";
        license = lib.licenses.agpl3Plus;
        hydraPlatforms = lib.platforms.none;
```

Comment:
You'll need to remove this line as well, otherwise hydra won't build the package.

Reply:
Shouldn't that be done on master first?

Reply:
This line is not present on master, as the file was regenerated before the merge from haskell-updates.